scripts/version.py

```diff
@@ -70,28 +73,23 @@ def build_number():
     # We can't go backwards. So we're adding to the one submitted at first.
     build_base_number = 2008998000
 
-    buildkite_build_number = os.getenv('BUILDKITE_BUILD_NUMBER')
-    increment_for_64bit = 1 if os.getenv('ARCH', '') == '64bit' else 0
-    
-    print('--- Assigning Build Number')
+    buildkite_build_number = os.getenv("BUILDKITE_BUILD_NUMBER")
+    increment_for_64bit = 1 if os.getenv("ARCH", "") == "64bit" else 0
 
     if buildkite_build_number is not None:
-        build_number = build_base_number + 2 * int(buildkite_build_number) + increment_for_64bit
-        print(build_number)
+        build_number = (
+            build_base_number + 2 * int(buildkite_build_number) + increment_for_64bit
```

Comment:
oof.

I recall some recent discussions around changing this up. We holding off on that? 

Also, this still depends on `BUILDKITE_BUILD_NUMBER` - are we setting that manually via GHA? Or are we skipping this bit altogether? Noting that this is how Android determines whether or not to "upgrade" and installed app.

Reply:
At the moment I'm only building the debug build in Github Actions - resolving this seemed like a blocker to being able to build the non-debug but develop builds for PRs.